CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- FIX: do not try cast entity id and entity type in update rules when expand parameters of rule to avoid BadRequest en CB tries to update that entities (#791)
```

Comment:
```suggestion
- Fix: do not try cast entity id and entity type in update rules when expand parameters of rule to avoid BadRequest en CB tries to update that entities (#791)
```